bdk_electrum_example/src/main.rs

```diff
@@ -23,13 +24,20 @@ enum ElectrumCommands {
     Sync {
         /// Scan all the unused addresses
         #[clap(long)]
```

Comment:
I would provide some defaults for these args

Reply:
The defaults are just "false" for now, we need to do the weird logic below, let's defer this!